projects/angular/src/emphasis/_labels.clarity.scss

```diff
@@ -110,84 +57,48 @@
   .label,
   a.label {
     //default
-    $label-nudge-for-vertical-alignment: variables.$clr_baselineRem_1px;
-    @include mixins.css-var(
-      font-size,
-      clr-label-font-size,
-      label-variables.$clr-label-font-size,
-      variables.$clr-use-custom-properties
-    );
-    @include mixins.css-var(
-      font-weight,
-      clr-label-font-weight,
-      label-variables.$clr-label-font-weight,
-      variables.$clr-use-custom-properties
-    );
-    @include mixins.css-var(
-      letter-spacing,
-      clr-label-letter-spacing,
-      label-variables.$clr-label-letter-spacing,
-      variables.$clr-use-custom-properties
-    );
-    @include mixins.css-var(
-      line-height,
-      clr-label-line-height,
-      // line-height needed to be adjusted to force center alignment
-      label-variables.$clr-label-font-size + variables.$clr_baselineRem_1px,
-      variables.$clr-use-custom-properties
-    );
+    @include mixins.generate-typography-token('CAPTION-LG-11');
 
     display: inline-flex;
     justify-content: center;
     align-items: center;
-    @include mixins.css-var(
-      padding,
-      clr-label-padding,
-      0 label-variables.$clr-label-padding-left-right 0,
-      variables.$clr-use-custom-properties
-    );
-    @include mixins.css-var(gap, clr-label-gap-size, 0, variables.$clr-use-custom-properties);
-    @include mixins.css-var(
-      border-radius,
-      clr-label-border-radius,
-      variables.$clr_baselineRem_0_5,
-      variables.$clr-use-custom-properties
-    );
-    @include mixins.css-var(
-      border-width,
-      clr-label-border-width,
-      label-variables.$clr-label-borderwidth,
-      variables.$clr-use-custom-properties
-    );
+    padding: label-variables.$clr-label-padding;
+    gap: label-variables.$clr-label-gap-size;
```

Comment:
gap and all these could be removed 

![image](https://github.com/vmware-clarity/ng-clarity/assets/132376042/39a79830-0568-496c-9279-27dbaf1008ed)